### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by siddhant2202**
Updated README Document with correcting all grammatical error using Grammarly

## Standards checklist:

- [ ] The PR title is descriptive.
- [ ] The PR doesn't replicate another PR which is already open.
- [ ] I have read the contribution guide and followed all the instructions.
- [x] The code follows the code style guide detailed in the wiki.
- [ ] The code is mine or it's from somewhere with an MIT-compatible license.
- [ ] The code is efficient, to the best of my ability, and does not waste computer resources.
- [ ] The code is stable and I have tested it myself, to the best of my abilities.
- [ ] If the code introduces new aliases, I provide a valid use case for all plugin users down below.

## Changes:

- [...]

## Other comments:

...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ plugins=(
   git
   bundler
   dotenv
-  macos
```
Comment (LunarWatcher) [inline README.md:109]: No. `plugins` is case-sensitive

File: README.md [hunk 2]
```diff
@@ -1,12 +1,12 @@
 <p align="center"><img src="https://ohmyzsh.s3.amazonaws.com/omz-ansi-github.png" alt="Oh My Zsh"></p>
 
-Oh My Zsh is an open source, community-driven framework for managing your [zsh](https://www.zsh.org/) configuration.
+Oh My Zsh is an open-source, community-driven framework for managing your [zsh](https://www.zsh.org/) configuration.
 
 Sounds boring. Let's try again.
 
 **Oh My Zsh will not make you a 10x developer...but you may feel like one.**
 
-Once installed, your terminal shell will become the talk of the town _or your money back!_ With each keystroke in your command prompt, you'll take advantage of the hundreds of powerful plugins and beautiful themes. Strangers will come up to you in cafés and ask you, _"that is amazing! are you some sort of genius?"_
```
Comment (LunarWatcher) [inline README.md:9]: Spaces at the start of a quote isn't good grammar, nor an improvement

**Review by LunarWatcher — CHANGES_REQUESTED** [review]

**Comment by mcornella** [comment]
Can you apply the changes suggested?

**Comment by Clash-ion** [comment]
Can i take this issue?

**Comment by LunarWatcher** [comment]
This is a pull request, not an issue

**Pull request closed without merging**